test/socket_connect_test.cpp

```diff
@@ -45,11 +54,50 @@ TEST(Socket, Connectv4)
 TEST(Socket, Connectv6)
 {
 	Socket server6(Domain::IPv6, Type::TCP);
-	std::thread serverThread1(runServer, std::ref(server6), 15020);
+	std::thread serverThread1(runNonUnixServer, std::ref(server6), 15020);
 	serverThread1.detach();
 	std::this_thread::sleep_for(1s);
 
 	Socket client6(Domain::IPv6, Type::TCP);
 	ASSERT_NO_THROW(client6.connect("::1", 15020));
 	client6.close();
 }
+
+TEST(Socket, ConnectUnixTCP)
+{
+	std::string unixPathServer("/tmp/unixSocketFileServer7");
+	Socket serverUnixTCP(Domain::UNIX, Type::TCP);
+	std::thread serverThread1(runUnixServer, std::ref(serverUnixTCP),
+	                          &unixPathServer.front());
+	serverThread1.detach();
+	std::this_thread::sleep_for(2s);
+
+	std::string unixPathClient("/tmp/unixSocketFileClient7");
+	Socket clientUnixTCP(Domain::UNIX, Type::TCP);
+	EXPECT_NO_THROW(clientUnixTCP.bind([&](AddrUnix &s) {
+		return methods::construct(s, &unixPathClient.front());
+	}));
+	EXPECT_NO_THROW(clientUnixTCP.connect([&](AddrUnix &s) {
+		return methods::construct(s, &unixPathServer.front());
+	}););
+	clientUnixTCP.close();
```

Comment:
shouldn't need to explicitly call `close()` here, ending `}` is on next line only.